health/healthcheck.go

```diff
@@ -43,11 +73,30 @@ func (service *HealthService) annotationsAPICheck() fthealth.Check {
 
 func (service *HealthService) annotationsAPIChecker() (string, error) {
 	if err := service.annotationsAPI.GTG(); err != nil {
-		return "UPP Public Annotations API is not healthy", err
+		return "", err
```

Comment:
Why did the message text get removed?

Reply:
Because the string is never used by the health check  library in case of error.

Reply:
😞 You are right.

Reply:
Is that reason to remove it though? What if a future version of the lib makes use of it?